core/domain/suggestion_registry.py

```diff
@@ -354,7 +354,11 @@ def __init__(
         self.change = exp_domain.ExplorationChange(change)
         self.score_category = score_category
         self.last_updated = last_updated
+        # Suggestions of this type are not queryable by language code.
         self.language_code = None
+        # Currently, we don't allow adding images in the edit state content
```

Comment:
As mentioned elsewhere, put quotes around "edit state content".

Please fix this everywhere else.